packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/portabilities.less

```diff
@@ -2,49 +2,7 @@
 @import 'ovh-ui-kit-bs/dist/less/bootstrap/_variables';
 
 .telephony-portabilities {
-  li {
-    position: relative;
-    min-height: 50px;
-
-    &::before {
-      content: '';
-      position: absolute;
-      left: -16px;
-      border-left: 3px solid @gray-lighter;
-      height: 100%;
-      width: 1px;
-    }
-
-    &::after {
-      content: '';
-      display: block;
-      border-radius: 10px;
-      width: 20px;
-      height: 20px;
-      position: absolute;
-      left: -25px;
-      top: 17px;
-      background-color: @gray-lighter;
-    }
-
-    &.doing::after {
-      background-color: #3f5167;
-    }
-
-    &.done::after {
-      background-color: @as-500;
-    }
-
-    &.error::after {
-      background-color: @ae-500;
-    }
-
-    &:first-child::before {
-      top: 20px;
-    }
-
-    &:last-child::before {
-      height: 0;
-    }
+  .oui-collapsible__header {
+    height: auto;
```

Comment:
**question:** Is it required? Isn't `auto` the default value?

Reply:
This is necessary for mobile display.
Because I use collasible custom because the UI-kit one we can't put html in the heading